CONTRIBUTING.md

```diff
@@ -84,9 +83,13 @@ cd packages/ember-toucan-core
 pnpm start
 # ember-toucan-core is now in watch mode and will rebuild automatically
 
-# In a second terminal
-cd docs-app
-pnpm start
+# In a second terminal, cd to the root of the repo
+cd root-of-repo-where-that-is-for-you
+pnpm docs:start # using turbo here
+
+# In a third terminal, switch to the test-app
+cd root-of-repo-where-that-is-for-you/test-app
+pnpm start # and then navigate to /tests
```

Comment:
Should we add a comment above here about running `ember-toucan-form` in watch mode as well?  Or maybe that'd be a separate section?

Reply:
Let me add it

Reply:
Should be here now...around likes 84/85